packages/dicomImageLoader/src/imageLoader/isColorConversionRequired.ts

```diff
@@ -8,26 +8,32 @@
  * @param RGBA
  * @returns
  */
-export default function isColorConversionRequired(imageFrame, RGBA) {
+export default function isColorConversionRequired(imageFrame, transferSyntax) {
   if (imageFrame === undefined) {
     return false;
   }
   const { rows, columns, photometricInterpretation, pixelDataLength } =
     imageFrame;
 
+  // if it is jpeg don't convert
+  if (transferSyntax === '1.2.840.10008.1.2.4.50') {
+    return false;
+  }
+
   if (photometricInterpretation.endsWith('420')) {
     return (
-      pixelDataLength !==
+      pixelDataLength ===
       (3 * Math.ceil(columns / 2) + Math.floor(columns / 2)) * rows
     );
   } else if (photometricInterpretation.endsWith('422')) {
     return (
-      pixelDataLength !==
+      pixelDataLength ===
       (3 * Math.ceil(columns / 2) + Math.floor(columns / 2)) *
         Math.ceil(rows / 2) +
         Math.floor(rows / 2) * columns
     );
   } else {
```

Comment:
Could you add a test for RGBA here - we NEVER convert if it is RGBA already (well, maybe from RGBA to RGB, but that isn't considered a conversion).   pixelDataLength=== 4* columns*rows is a test for that.  Then, exclude the JPEG test, as that always results in RGBA.